src/psql.js

```diff
@@ -0,0 +1,46 @@
+import { log, error, execPromise, spawnPromise, Spinner } from "../utils";
+const queries = require("../lib/inquirer");
+const getUsernameDBname = queries.getUsernameDBname;
+
+const psql = {
+  psqlCLI: async () => {
+    const answers = await getUsernameDBname();
+    new Spinner(log("Launching the Postgres CLI. Please wait..."));
+
+    const cmd =
+      `docker exec -it postgres psql --username=${answers.username} ` +
+      `--dbname=${answers.dbname}`;
+    const launchPsql = spawnPromise(cmd, {
+      stdio: "inherit",
+      shell: true,
+    });
+    launchPsql.catch((result) => {
+      error(`ERROR: ${result.stderr}`);
+    });
+  },
+};
+
+// const ksql = {
```

Comment:
Testing this single line comment to see if there is a way to remove code directly in github.